components/patient-data/ui/src/main/resources/PhenoTips/PatientSheetCode.xml

```diff
@@ -735,10 +735,20 @@ document.observe('xwiki:dom:loaded', function() {
              if (forceUpdate) {
                 this._silentRemove(inputElt.value, type);
              }
+             var group = this._getGroup(sectionId);
              var exists = this._find(inputElt.value, type);
-             if (!exists) {
-               var group = this._getGroup(sectionId);
+             var existsDifferentGroup = false;
+             if (exists) {
+               var existsGroup = exists.up('.summary-group');
+               existsDifferentGroup = group != existsGroup;
+             }
+
+             if (!exists || existsDifferentGroup) {
                if (!group) {return;}
+               if (existsDifferentGroup) {
+                 // The section id of the `exists` variable should ideally be passed into `_silentRemove`.
+                 this._silentRemove(inputElt.value, type)
+               }
```

Comment:
If there is to be no ```_silentRemove``` here, then when someone adds the same phenotype in two different sections it will appear twice in the summary (under different sections).
I wounder if that is the better approach.